webpack.config.js

```diff
@@ -3,6 +3,9 @@ const HtmlWebpackPlugin = require('html-webpack-plugin')
 const webpack = require('webpack')
 const BuildExportsList = require('./lib/BuildExportsList')
 
+const hotMiddlewareScript =
+  'node_modules/webpack-hot-middleware/client?path=/__webpack_hmr&timeout=20000&reload=true'
```

Comment:
When we start publishing this I want to add `&quiet=true` to this so that the dev using it does not get informed about our warnings or errors.